jans-cedarling/cedarling/src/common/cedar_schema.rs

```diff
@@ -184,7 +184,7 @@ impl<'de> serde::Deserialize<'de> for CedarSchema {
 mod deserialize {
     #[derive(Debug, thiserror::Error)]
     pub enum ParseCedarSchemaSetMessage {
-        #[error("unable to decode cedar policy schema base64")]
+        #[error("Failed to decode Base64 encoded string")]
```

Comment:
it would be great to extend
```suggestion
        #[error("Failed to decode Base64 encoded string cedar policy schema")]
```